### PR TITLE
Implement readRaw for a few endpoints

### PR DESCRIPTION
**Pull request by haubitzer**

### CODE REVIEW

File: src/generalSettings/index.ts [hunk 1]
```diff
@@ -12,6 +13,12 @@ export default class GeneralSettings extends BaseApi {
     return responseHandler.requiredList(response);
   }
 
+  public readRaw(requestParams?: RequestParams<GeneralSetting> | Object): Promise<ApiResponseOnSuccess<GeneralSetting[]>> {
+    const params = requestParams instanceof RequestParams ? requestParams.getParams() : requestParams;
+    const response = this._get<GeneralSetting>(`${this.getResourceName()}/read`, { params });
```
Comment (benj56) [inline src/generalSettings/index.ts:18]:
```suggestion
    const response = this._get<GeneralSetting[]>(`${this.getResourceName()}/read`, { params });
```
This shouldn't compile without the array, but with the messed up types we have now it does. Should be fixed with #33 . I hope I'll have the time to get that PR ready in the next days

File: src/projects/index.ts [hunk 2]
```diff
@@ -11,6 +12,11 @@ export default class Projects extends BaseApi {
     const response = this._get<Project>(`${this.getResourceName()}/read`, { params });
     return responseHandler.requiredList(response);
   }
+  public readRaw(requestParams?: RequestParams<Project> | Object): Promise<ApiResponseOnSuccess<Project[]>> {
+    const params = requestParams instanceof RequestParams ? requestParams.getParams() : requestParams;
+    const response = this._get<Project>(`${this.getResourceName()}/read`, { params });
```
Comment (benj56) [inline src/projects/index.ts:17]:
```suggestion
    const response = this._get<Project[]>(`${this.getResourceName()}/read`, { params });
```

File: src/absenceTypes/absenceTypes.test.ts [hunk 3]
```diff
@@ -3,18 +3,21 @@ import { AbsenceType } from './types';
...
   var mock = new AxiosMockAdapter(axios);
   var result: Promise<AbsenceType[]> | null;
   var resultSingle: Promise<AbsenceType> | null;
+  var resultRaw: Promise<ApiResponseOnSuccess<AbsenceType[]>> | null;
 
   afterEach(() => {
     mock.reset();
     result = null;
     resultSingle = null;
+    resultRaw = null;
```
Comment (benj56) [inline src/absenceTypes/absenceTypes.test.ts:20]: I would remove those vars instead of adding another one, but maybe it also makes sense to do that in a separate PR for all the endpoints at once

**Review by benj56 — DISMISSED** [review]

File: src/absenceTypes/absenceTypes.test.ts (hunk 3 above)
Comment (haubitzer) [reply src/absenceTypes/absenceTypes.test.ts:20]: yes, we will change this in a separate PR

File: src/generalSettings/index.ts (hunk 1 above)
Comment (haubitzer) [reply src/generalSettings/index.ts:18]: fixed now only for readRaw, other cases will be fixed in #33

File: src/projects/index.ts (hunk 2 above)
Comment (haubitzer) [reply src/projects/index.ts]: fixed

**Review by benj56 — APPROVED** [review]
